wasm.md

```diff
@@ -600,26 +601,58 @@ Here, we allow for an "abstract" function declaration using syntax `func_::___`,
 
     syntax FuncDecl  ::= "(" FuncDecl ")"     [bracket]
                        | TypeKeyWord ValTypes
-                       | "export" Identifier
     syntax FuncDecls ::= List{FuncDecl, ""} [klabel(listFuncDecl)]
  // --------------------------------------------------------------
 
-    syntax Instr ::= "(" "func"              FuncDecls Instrs ")"
-                   | "(" "func" Identifier FuncDecls Instrs ")"
-                   | "func" Identifier "::" FuncType VecType "{" Instrs "}"
- // -----------------------------------------------------------------------
-    rule <k> ( func FDECLS INSTRS )
-          => func gatherExportedName(FDECLS) :: gatherFuncType(FDECLS) gatherTypes(local, FDECLS) { INSTRS }
+
+    syntax FuncAbbr ::= "(" "export" String ")" FuncAbbr // `import`, if exists, should always be the last one. So should add subsort `| "(" "import" Identifier ")"` in the future.
```

Comment:
Put documentation in the markdown, not as comments in the K code.

Reply:
fixed